pkgs/development/coq-modules/LibHyps/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, mkCoqDerivation, coq, version ? null }:
+
+with lib;
+mkCoqDerivation {
+  pname = "LibHyps";
+  owner = "Matafou";
+  inherit version;
+  defaultVersion = if (versions.range "8.11" "8.15") coq.version then "2.0.4.1" else null;
+  release = {
+    "2.0.4.1".sha256 = "09p89701zhrfdmqlpxw3mziw8yylj1w1skb4b0xpbdwd1vsn4k3h";
+  };
+
+  configurePhase = "./configure.sh";
```

```suggestion
  configureScript = "./configure.sh";
```

That way, the `preConfigure` and `postConfigure` hooks are not ignored.

---

Thanks, I didn't know about `configureScript`.

---

Note that this was part of #157987. I've updated that one and rebased this one on top.